### ISSUE TITLE
HTTPS for citizencodeofconduct.org

### ISSUE THREAD

**Issue by annevk** [stumpsyn/policies#8]
Would be great if the domain could be served over HTTPS/HSTS, providing integrity of the content. Relatively easy now on a number of hosting providers that support Let's Encrypt.

**Comment by Hanylylyya97** [comment]
https://git-for-windows.github.io/#contribute

**Comment by billyjoebaron** [comment]
sooo im new could you dumb that down a little. Im still working on the lingo lol

**Comment by tonyalester** [comment]
ty

**Comment by annevk** [comment]
@gabelula assuming this is hosted through GitHub, would you mind enabling GitHub's new HTTPS option? Thanks!

**Comment by reidab** [comment]
It's sadly not hosted though GitHub pages, but we'll be setting up HTTPS on this domain soon.

**Comment by LooTKinGz** [comment]
Coc update

**Comment by annevk** [comment]
Is there anything I could do to help with this?
